azure-pipelines.yml

```diff
@@ -42,277 +33,239 @@ resources:
   containers:
   - container: LinuxContainer
     image: mcr.microsoft.com/dotnet-buildtools/prereqs:cbl-mariner-2.0-fpm
+  repositories:
+  - repository: 1ESPipelineTemplates
+    type: git
+    name: 1ESPipelineTemplates/1ESPipelineTemplates
+    ref: refs/tags/release
 
-stages:
-- stage: build
-  displayName: Build
-  jobs:
-  - template: /eng/common/templates/jobs/jobs.yml
-    parameters:
-      enableMicrobuild: true
-      enablePublishBuildArtifacts: true
-      enablePublishBuildAssets: true
-      enablePublishUsingPipelines: ${{ variables._PublishUsingPipelines }}
-      enableTelemetry: true
-      enableSourceBuild: true
-      helixRepo: dotnet/arcade-validation
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1ESPipelineTemplates
+  parameters:
+    pool:
+      name: $(PoolProvider)
+      image: windows.vs2022.amd64
+      os: windows
+    featureFlags:
+      autoBaseline: true
```

Comment:
do we want to check auto-baselining in❔ my intention w/ dotnet-helix-machines was to baseline once and then restore TSA settings. that would prevent failing builds but keep a record of any issues found